Packs/DNSDB/Integrations/DNSDB/README.md

```diff
@@ -0,0 +1,25 @@
+# Farsight DNSDB
+
+This integration uses Farsight Security’s DNSDB solution to interactively lookup rich, historical DNS information – either as playbook tasks or through API calls in the War Room – to access rdata and rrset records.
+
+1. Go to 'Settings > Integrations > Servers & Services'
+1. Locate the DNSDB integration by searching for 'Farsight DNSDB' using the search box on the top of the page.
+1. Click 'Add instance' to create and configure a new integration. You should configure the following DNSDB and Demisto specific settings:
+
+    Name
+    :    A textual name for the integration instance.
+    
+    API Key
+    :    The API key that user gets from Farsight Security.
+    
+    DNSDB Service URL
+    :    The service URL for Farsight DNSDB.
+    
+    Use system proxy settings
+    :    Select whether or not to communicate via the system proxy server.
+    
+    Demisto engine
+    :    If relevant, select the engine that acts as a proxy to the server. [Engines](https://support.demisto.com/hc/en-us/articles/226274727-Settings-Integrations-Engines) are used when you need to access a remote network segments and there are network devices such as proxies, firewalls, etc. that prevent the Demisto server from accessing the remote networks.
+    
+1. Press the 'Test' button to validate connection. If you are experiencing issues with the service configuration, please contact [Farsight Security support](support@farsightsecurity.com)
+1. After completing the test successfully, press the 'Done' button.
```

Comment:
9e24c3a4d

Is there anything to help generate the examples and output?

Reply:
Yes by using the -e flag I wrote in my previous comment. From the documentation:
```  
-e, --examples Path for a file containing command or script examples. Each Command should be in a separate line. For script - the script example surrounded by double-quotes.
```
This will go to your designated demisto server and will run each command with the command example. Search for the HellowWorls pack in your local repo and inside `Packs/HelloWorld/Integrations/HelloWorld/command_examples` you will see an example.